tests/test_virtual_chassis.py

```diff
@@ -906,9 +904,8 @@ def test_chassis_add_remove_ports(self, vct):
                 assert len(num) == num_ports
 
                 # Check that we see the logs for removing default vlan
-                matching_log = "removeDefaultVlanMembers: Remove 0 VLAN members from default VLAN"
                 _, logSeen = dvs.runcmd( [ "sh", "-c",
-                     "awk '/{}/,ENDFILE {{print;}}' /var/log/syslog | grep '{}' | wc -l".format( marker, matching_log ) ] )
+                    "awk STARTFILE/ENDFILE /var/log/syslog | grep 'removeDefaultVlanMembers: Remove 32 VLAN members from default VLAN' | wc -l"] )
```

Comment:
why do we need to change the matching log here?

Reply:
We have updated system ports format since it wasn't correct before. After this change, we have seen syslog message got updated. I checked with @arista-nwolfe  and he confirmed above that the new syslog is ok.